view/NFTView.tsx

```diff
@@ -68,94 +69,67 @@ interface NFTItem {
   amount: number;
   status: string;
 }
-
 export default function NFTView() {
-  const [selectQuery, setSelectQuery] =
-    useState<SelectOption[]>(FILTER_CONTENTS);
+  const [selectQuery, setSelectQuery] = useState<SelectOption[]>(FILTER_CONTENTS);
```

Comment:
query보다는 options가 더 괜찮을 것 같아요~

```suggestion
  const [selectOptions, setSelectOptions] = useState<SelectOption[]>(FILTER_CONTENTS);
```